packages/core/admin/admin/src/content-manager/pages/ListView/components/AdminUsersFilter.tsx

```diff
@@ -1,3 +1,4 @@
+import React from 'react';
```

Comment:
```suggestion
import * as React from 'react';
```
it is the common way we use to import React

Reply:
Done